src/plugins/index_pattern_management/public/components/edit_index_pattern/scripted_fields_table/scripted_fields_table.tsx

```diff
@@ -145,15 +147,12 @@ export class ScriptedFieldsTable extends Component<
       <>
         <Header
           addScriptedFieldUrl={`${window.location.origin +
-            window.location.pathname}#/management/kibana/index_patterns/${
+            window.location.pathname}${INDEX_PATTERN_MANAGEMENT_SECTION_PATH}/patterns/${
```

Comment:
This needs to use react-router-dom and a relative url to navigate

Reply:
done